source/vsts.md

```diff
@@ -96,7 +97,7 @@ Options include:
  * **Package Version**: The version of the package, pushed separately, to push build information onto.
  * **Work Items Source**: The service hosting any work items or issues associated with each version of the package. Octopus will add information about the work items or issues to the package build information, which can be used in release notes. For more information see the [Issue Trackers documentation](https://octopus.com/docs/deployment-process/issue-tracking).
  * **Replace Existing**: If the package build information already exists in the repository, the default behavior is to reject the new build information being pushed. Set this flag to 'True' to overwrite the existing package build information.
- * **Additional Arguments**: Any additional [Octo arguments](https://octopus.com/docs/octopus-rest-api/octo.exe-command-line/push-metadata) to include.
+ * **Additional Arguments**: Any additional [Octopus CLI arguments](https://octopus.com/docs/octopus-rest-api/octopus-cli/push-metadata) to include.
```

Comment:
This links to the deprecated metadata doc, rather than the Build Information doc